src/main/resources/assets/controlify/lang/fr_fr.json

```diff
@@ -0,0 +1,393 @@
+
+  "controlify.battery_level.unknown": "Inconnu",
+  "controlify.battery_level.empty": "Vide",
+  "controlify.battery_level.low": "Faible",
+  "controlify.battery_level.medium": "Moyen",
+  "controlify.battery_level.high": "Élevé",
+  "controlify.battery_level.full": "Plein"
+
+  "controlify.command.vibratecontroller.static.single": "Contrôleur vibré d'1 joueur.",
+  "controlify.command.vibratecontroller.static.multiple": "Contrôleurs vibrés de %s joueurs.",
+  "controlify.command.vibratecontroller.pos.single": "Contrôleur vibré d'1 joueur à %s, %s, %s.",
+  "controlify.command.vibratecontroller.pos.multiple": "Contrôleurs vibrés de %s joueurs à %s, %s, %s.",
+  "controlify.command.vibratecontroller.entity.single": "Contrôleur vibré d'1 joueur depuis la position de %s.",
+  "controlify.command.vibratecontroller.entity.multiple": "Contrôleurs vibrés de %s joueurs depuis la position de %s."
+
+  "controlify.compat.svc.hold": "(Maintenir)",
+  "controlify.compat.svc.toggle": "(Basculer)",
+
+  "controlify.hat_state.up": "Haut",
+  "controlify.hat_state.down": "Bas",
+  "controlify.hat_state.left": "Gauche",
+  "controlify.hat_state.right": "Droite",
+
+  "controlify.joystick_mapping.unmapped.axis": "Axe #%s",
+  "controlify.joystick_mapping.unmapped.button": "Bouton #%s",
+  "controlify.joystick_mapping.unmapped.hat": "#%s",
+  "controlify.joystick_mapping.unmapped.axis_direction.negative": "(Négatif)",
+  "controlify.joystick_mapping.unmapped.axis_direction.positive": "(Positif)"
+
+
+
+  "controlify.joystick_mapping.tm_t16000.axis.roll": "Roll",
+  "controlify.joystick_mapping.tm_t16000.axis.pitch": "Pitch",
+  "controlify.joystick_mapping.tm_t16000.axis.yaw": "Yaw",
+  "controlify.joystick_mapping.tm_t16000.axis.throttle": "Throttle",
+  "controlify.joystick_mapping.tm_t16000.button.trigger": "Trigger",
+  "controlify.joystick_mapping.tm_t16000.button.a": "A",
+  "controlify.joystick_mapping.tm_t16000.button.x": "X",
+  "controlify.joystick_mapping.tm_t16000.button.b": "B",
+  "controlify.joystick_mapping.tm_t16000.left_1": "Gauche 1",
+  "controlify.joystick_mapping.tm_t16000.left_2": "Gauche 2",
+  "controlify.joystick_mapping.tm_t16000.left_3": "Gauche 3",
+  "controlify.joystick_mapping.tm_t16000.left_4": "Gauche 4",
+  "controlify.joystick_mapping.tm_t16000.left_5": "Gauche 5",
+  "controlify.joystick_mapping.tm_t16000.left_6": "Gauche 6",
+  "controlify.joystick_mapping.tm_t16000.right_1": "Droite 1",
+  "controlify.joystick_mapping.tm_t16000.right_2": "Droite 2",
+  "controlify.joystick_mapping.tm_t16000.right_3": "Droite 3",
+  "controlify.joystick_mapping.tm_t16000.right_4": "Droite 4",
+  "controlify.joystick_mapping.tm_t16000.right_5": "Droite 5",
+  "controlify.joystick_mapping.tm_t16000.right_6": "Droite 6",
+  "controlify.joystick_mapping.tm_t16000.hat": "Hat"
```

Comment:
Chapeau ?  Chapeau chinois ?  "bouton de « point de vue »" ?

Reply:
I'm not sure about this translation, I think I'll leave it untranslated.

Reply:
Fair enough, "hat" is a technical term that should be understood by a HOTAS owner ;)